post-scan-actions/aws-python-promote-or-quarantine/terraform/variables.tf

```diff
@@ -0,0 +1,63 @@
+# Configure the variables here
+variable "region" {
+  type = string
+  description = "AWS region"
+  default = "us-east-1"
+  validation {
+    condition     = can(regex("[a-z][a-z]-[a-z]+-[1-2]", var.region))
+    error_message = "Must be valid AWS Region names."
+  }
+ }
+
+variable "ScanningBucketName" {
+  type = string
+  default = ""
+  description = "The S3 bucket scanned by Trend Micro Cloud One File Storage Security."
+ }
+
+variable "ScanResultTopicARN" {
+  type = string
+  default = ""
+  description = "The ARN of the scan result SNS topic in storage stack."
+ }
+
+variable "PromoteBucketName" {
+  type = string
+  default = ""
+  description = "[Optional] The bucket name used to promote files without malicious intent. Leave the bucket name blank to disable promoting."
+ }
+
+variable "PromoteMode" {
+  type = string
+  description = "The method by which files were promoted. (Options: move, copy)"
+  default = "move"
+  validation {
+    condition = contains(["move", "copy"], var.PromoteMode)
+    error_message = "The promote mode must be either move or copy."
+  }
+ }
+
+variable "QuarantineBucketName" {
+  type = string
+  default = ""
+  description = "[Optional] The bucket name to quarantine malicious files. Leave the bucket name blank to disable quarantining"
+ }
+
+ variable "QuarantineMode" {
+  type = string
+  description = "The method by which files were quarantined. (Options: move, copy)"
+  default = "move"
+  validation {
+    condition = contains(["move", "copy"], var.QuarantineMode)
+    error_message = "The quarantine mode must be either move or copy."
+  }
+ }
+ 
+variable "ACL" {
```

Comment:
Validate the value of  `ACL` by the options.

Reply:
That has been done by one of the commit suggestions.